scripts/delayed.js

```diff
@@ -1,8 +1,5 @@
 // eslint-disable-next-line import/no-cycle
-import { loadScript, sampleRUM } from './aem.js';
-
-// Core Web Vitals RUM collection
-sampleRUM('cwv');
```

Comment:
Why has this been removed?

Reply:
The call to `sampleRUM.enhance` (which you'll note in latest changes occurs in `updateSectionStatus`) loads what we call the rum enhancer, see https://github.com/adobe/helix-rum-enhancer

That takes care of all the collection calls removed in this PR, without customers having to update their codebases as we make rum collection changes. So this is a one time update to get you on rum v2 using the new rum enhancer, but as we add more features to what rum we collect, no future changes should be needed on your part